Sources/View/ViewModelView.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  ViewModelView.swift
+//  Alicerce
+//
+//  Created by André Pacheco Neves on 05/05/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public protocol ViewModelView: View {
+    associatedtype ViewModel
+
```

Comment:
i see your point, it makes a lot of sense, but it is developer responsibility. This is a base for something, not a prison 😄 right?